### ISSUE TITLE
[BUG]: [ERROR] 404: Not Found

### ISSUE THREAD

**Issue by napgiver** [anidl/multi-downloader-nx#731]
### Program version

5.1.3

### Operating System

Windows

### Type

GUI

### Service

Crunchyroll

### Command used

Download

### Show ID

G1XHJV08M

### Episode

12

### Console Output

```Shell
=== Multi Downloader NX GUI 5.1.3 ===

GUI server started on port 3000
[WS] Connection to public ws from '::ffff:127.0.0.1'
[WS] Connection from '::ffff:127.0.0.1'
[WS] Connection from '::ffff:127.0.0.1'
USER: PRIVATE (PRIVATE@gmail.com)
Your Country: US

USER: PRIVATE (PRIVATE@gmail.com)
Your Country: US
USER: PRIVATE (PRIVATE@gmail.com)
USER: PRIVATE (PRIVATE@gmail.com)
[DEBUG] Got search options: {"search":"Spice and Wolf","search-type":"series"}
Found series:
  [Z:G6GG38246|SRZ.275311] Spice and Wolf (Seasons: 2, EPs: 25) [SUB, DUB]
    - Subtitles: en, es-419, pt-BR
    [S:G6P8CX55Z] Spice and Wolf (Season: 1) [SUB, DUB]
      - Versions: en, es-419, pt-BR, ja
      - Subtitles: en, es-419, pt-BR
    [S:GY09N3JGY] Spice and Wolf II (Season: 2) [SUB, DUB]
      - Versions: en, es-419, pt-BR, ja
      - Subtitles: en, es-419, pt-BR

  [Z:G1XHJV08M|SRZ.283817] Spice and Wolf: MERCHANT MEETS THE WISE WOLF (Seasons: 1, EPs: 15) [SIMULCAST, SUB, DUB]
    - Subtitles: en, es-419, es-ES, pt-BR, fr, de, ar, it, ru
    [S:GRMGC3K3X] Spice and Wolf: MERCHANT MEETS THE WISE WOLF (Season: 1) [SIMULCAST, SUB, DUB]
      - Versions: en, fr, de, ja
      - Subtitles: en, es-419, es-ES, pt-BR, fr, de, ar, it, ru

  [Z:G4PH0WZ1K|SRZ.283074] Wolf Children (Seasons: 1, EPs: 1) [SUB, DUB]
    - Subtitles: en
    [S:GYMGC3W9P] Wolf Children (Season: 1) [DUB]
      - Versions: en
      - Subtitles: en

  [Z:G4PH0WMEX|SRZ.281985] High Guardian Spice (Seasons: 1, EPs: 12) [DUB]
    - Subtitles: en, es-419, es-ES, pt-BR, fr, de, it, ru
    [S:G6Q4CZD4P] High Guardian Spice (Season: 1) [DUB]
      - Versions: en
      - Subtitles: en, es-419, es-ES, pt-BR, fr, de, it, ru

  [Z:GY1XMV90Y|SRZ.274547] Wolf's Rain (Seasons: 1, EPs: 26) [SUB, DUB]
    - Subtitles: en, es-419, pt-BR
    [S:GRJ059Q2Y] Wolf's Rain (Season: 1) [SUB, DUB]
      - Versions: en, ja
      - Subtitles: en, es-419, pt-BR

  Total results: 1609 (Page: 1/322)
USER: PRIVATE (PRIVATE@gmail.com)
[E1] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - The Harvest Festival and The Crowded Driver's Box [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E2] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Mischievous Wolf and No Laughing Matter [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E3] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Port Town and Sweet Temptation [☆ Japanese, ☆ English, ☆ French, ☆ German]
[E4] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Romantic Merchant and Moonlit Farewell [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E5] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wolf Incarnate and Obedient Lamb [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E6] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Merchant and Unreasonable God [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E7] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Scale of God and Sorcerer of the Grasslands [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E8] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Fellow Traveler and Foreboding News [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E9] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Sweet Honey and Bitter Armor [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E10] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wisdom of a Wolf and Smooth Talk of a Merchant [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E11] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Forest of Wolves and Frigid Rain [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E12] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Price of Betrayal and Price of Gold [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E13] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Supper of Three and Afternoon of Two [☆ Japanese]
[E14] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - New Town and Nostalgic Feeling [☆ Japanese]
[E15] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Bird Feather and Mysterious Ore [☆ Japanese]
USER: PRIVATE (PRIVATE@gmail.com)
[DEBUG] Got resolve options: {"id":"G1XHJV08M","q":4,"e":",12","dubLang":["eng"],"dlsubs":["en"],"fileName":"${seriesTitle} S${season}E${episode} ${title}","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false}
[E1] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - The Harvest Festival and The Crowded Driver's Box [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E2] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Mischievous Wolf and No Laughing Matter [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E3] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Port Town and Sweet Temptation [☆ Japanese, ☆ English, ☆ French, ☆ German]
[E4] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Romantic Merchant and Moonlit Farewell [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E5] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wolf Incarnate and Obedient Lamb [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E6] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Merchant and Unreasonable God [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E7] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Scale of God and Sorcerer of the Grasslands [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E8] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Fellow Traveler and Foreboding News [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E9] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Sweet Honey and Bitter Armor [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E10] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wisdom of a Wolf and Smooth Talk of a Merchant [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E11] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Forest of Wolves and Frigid Rain [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E12] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Price of Betrayal and Price of Gold [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E13] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Supper of Three and Afternoon of Two [☆ Japanese]
[E14] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - New Town and Nostalgic Feeling [☆ Japanese]
[E15] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Bird Feather and Mysterious Ore [☆ Japanese]

------------------------------

[S1E12] - Price of Betrayal and Price of Gold [✓ English]
USER: PRIVATE (PRIVATE@gmail.com)
[DEBUG] Got download options: {"id":"G1XHJV08M","q":4,"e":"12","dubLang":["eng"],"dlsubs":["en"],"fileName":"${seriesTitle} S${season}E${episode} ${title}","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false,"ids":["G8WUN110P"],"title":"Price of Betrayal and Price of Gold","parent":{"title":"Spice and Wolf: MERCHANT MEETS THE WISE WOLF","season":"1"},"image":"https://www.crunchyroll.com/imgsrv/display/thumbnail/1200x675/catalog/crunchyroll/01a784cf7ea16056a13cd1405f6fc704.jpg","episode":"12"}
[E1] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - The Harvest Festival and The Crowded Driver's Box [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E2] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Mischievous Wolf and No Laughing Matter [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E3] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Port Town and Sweet Temptation [☆ Japanese, ☆ English, ☆ French, ☆ German]
[E4] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Romantic Merchant and Moonlit Farewell [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E5] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wolf Incarnate and Obedient Lamb [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E6] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Merchant and Unreasonable God [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E7] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Scale of God and Sorcerer of the Grasslands [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E8] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Fellow Traveler and Foreboding News [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E9] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Sweet Honey and Bitter Armor [☆ Japanese, ☆ English, ☆ German, ☆ French]
[E10] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Wisdom of a Wolf and Smooth Talk of a Merchant [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E11] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Forest of Wolves and Frigid Rain [☆ Japanese, ☆ German, ☆ French, ☆ English]
[E12] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Price of Betrayal and Price of Gold [☆ Japanese, ☆ German, ☆ English, ☆ French]
[E13] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Supper of Three and Afternoon of Two [☆ Japanese]
[E14] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - New Town and Nostalgic Feeling [☆ Japanese]
[E15] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - Season 1 - Bird Feather and Mysterious Ore [☆ Japanese]

------------------------------

[S1E12] - Price of Betrayal and Price of Gold [✓ English]
Requesting: [G8WUN110P] Spice and Wolf: MERCHANT MEETS THE WISE WOLF - 12 - Price of Betrayal and Price of Gold
Chapter request successful
[ERROR] 404: Not Found
[ERROR] {"code":"content.error.route_not_found","context":[]}
[ERROR] Request Stream URLs FAILED! Attempting fallback
[ERROR] 404: Not Found
[ERROR] {"code":"content.error.route_not_found","context":[]}
[ERROR] Fallback Request Stream URLs FAILED!
[ERROR] Download error: Unable to download episode 12 from G1XHJV08M
```

### Additional Information

Everything was working just fine yesterday. Just to check, I tried to grab E11, which I had no problems getting last week, but, now it's giving me the same error as E12. It's not limited to _Spice and Wolf: MERCHANT MEETS THE WISE WOLF._ I get this error on every show now.

**Comment by AnimeDL** [comment]
Should be fixed in the latest release, 5.1.4. They completely removed the old API that we were still using for some metadata, so I had to adapt to completely using the new API

**Comment by napgiver** [comment]
Yep, That fixed it. Thanks 👍
